05-html-css/css-framework.md

```diff
@@ -18,185 +16,144 @@ The usual components are:
-<<-'RUBY'
+...
 
-@import 'motion-ui/motion-ui';
-@include motion-ui-transitions;
-@include motion-ui-animations;
+append_to_file 'app/assets/stylesheets/application.css' do
+  <<-'SCSS'
 
-RUBY
+/* Custom bootstrap variables must be set or imported *before* bootstrap. */
+@import "bootstrap";
+/* Import scss content */
+@import "**/*";
+  SCSS
 end
 ```
 
 These foundation template changes will cause Rails to:
-1. Add the `foundation-rails` gem to the gemfile.  
-2. Run a generator command to work with the gem to generate a bunch of CSS and JavaScript files.  
-3. Lastly add some code to `foundation_and_overrides.scss` in the stylesheets folder to include some libraries for animated menus and transitions.
-
-Now when you create a new Rails project you will see the following stylesheets.
-
-![Foundation Gem Stylesheets](imgs/foundation_stylesheets.png)
-
-You can change foundation settings by editing the `_settings.scss` file and turn off and on Foundation features by changing the `foundation_and_overrides.scss` file.  You can also override foundation styles, by defining css rules of your own in the `application.css` file.  
+1. Add the `bootstrap` gems to the gemfile.
+2. Tell Rails to import some JavaScript and CSS files for Bootstrap.
 
-## Using Foundation Typography
+## Using Bootstrap Typography
 
 You should first notice that foundation changes the default styling of all the header and paragraph text styling
 
-### Before Foundation:
+### Before Bootstrap:
 ![before-foundation-typography](imgs/before-foundation.png)
 
-### After Foundation:
-![after-foundation-typography](imgs/after-foundation.png)
+### After Bootstrap:
+![Bootstrap Typography](imgs/bootstrap-typography.png)
 
-As you can see above Foundation provides an set of default fonts and styles for the basic elements.  It also provides styling for lists, abbreviations, quotations and accessibility.  You can read about Foundation's typography [here](https://foundation.zurb.com/sites/docs/typography-base.html).
 
-## Built-in Foundation Classes
+As you can see above Bootstrap provides an set of default fonts and styles for the basic elements.  It also provides styling for lists, abbreviations, quotations and accessibility.  You can read about Bootstrap's typography [here](http://getbootstrap.com/docs/4.1/content/typography/).
 
-Foundation also provides a number pre-created stylings for class names you can add to common HTML elements and take advantage of ready-made stylings.  
+## Built-in Bootstrap Classes
+
+Bootstrap also provides a number pre-created stylings for class names you can add to common HTML elements and take advantage of ready-made stylings.
 
 For example, Foundation also provides a variety of ways to create buttons on your site either using anchor or button elements.  You can experiment with foundation buttons [here](https://codepen.io/adadev/pen/WzYvyo?editors=1000).
```

Comment:
This Codepen attached:

1. Is untitled accidentally
2. Definitely is still Foundation